misc/scripts/library-coverage/compare-files-comment-pr.py

```diff
@@ -41,15 +46,34 @@ def compare_files_str(file1, file2):
     return ret
 
 
-def comment_pr(folder1, folder2, output_file, repo, pr_number, run_id):
-    compare_folders(folder1, folder2, output_file)
+def write_diff_for_run(output_file, repo, run_id):
+    folder1 = "out_base"
+    folder2 = "out_merge"
+    try:
+        utils.subprocess_run(["gh", "run", "download", "--repo", repo, "--name",
+                              "csv-framework-coverage-base", "--dir", folder1, str(run_id)])
+        utils.subprocess_run(["gh", "run", "download", "--repo", repo, "--name",
+                              "csv-framework-coverage-merge", "--dir", folder2, str(run_id)])
+        utils.subprocess_run(["gh", "run", "download", "--repo", repo, "--name",
+                              "pr", "--dir", "pr", str(run_id)])
+
+        compare_folders(folder1, folder2, output_file)
```

Comment:
Would it be useful to archive the diff as a run artifact, so that you don't have to make 3 API calls here and re-run the comparison?

Reply:
Oh yes, it would. I'm changing it. Thanks.